.env.sample

```diff
@@ -0,0 +1,4 @@
+NYPL_API_BASE_URL
```

Comment:
Do you need something like `NYPL_API_BASE_URL=example/URL`?

Reply:
The next iteration should include integration with the API Gateway which will use this variable. For now, the endpoint is not fully implemented and integrated with the gateway.